src/protocols/memory/ddr4.stanza

```diff
@@ -236,26 +232,19 @@ with:
   keyword-constructor => true
   constructor => #DDR4-Data-ACC-Constraint
 
-doc: \<DOC>
-Constructor for the DDR4 Data Link Constraint
-@param width Width of the DDR4 interface (enumerated type)
-@param diff-dqs-rs Differential Routing Structure constraints for DQS signals
-@param se-dq-rs Routing Structure constraints for DQ signals
-<DOC>
-public defn DDR4-Data-ACC-Constraint (
-    width:DDR4-width ) -> DDR4-Data-ACC-Constraint:
-  DDR4-Data-ACC-Constraint(
-    width = width
-  )
+; doc: \<DOC>
+; Constructor for the DDR4 Data Link Constraint
+; @param width Width of the DDR4 interface (enumerated type)
+; @param diff-dqs-rs Differential Routing Structure constraints for DQS signals
+; @param se-dq-rs Routing Structure constraints for DQ signals
+; <DOC>
+; public defn DDR4-Data-ACC-Constraint () -> DDR4-Data-ACC-Constraint:
+;   DDR4-Data-ACC-Constraint()
```

Comment:
Why not just remove this ?

Reply:
I will remove it in the next PR (for updating DDR3).